addons/food/functions/fnc_consume.sqf

```diff
@@ -0,0 +1,67 @@
+/*
+	File Name: fnc_consume.sqf
+	Author: Jak Keen
+*/
+
+#include "../script_component.hpp"
+
+private ["_player", "_mode"];
+
+_player = _this select 1;
+_mode = _this select 2;
+
+switch (_mode) do {
+    case "eat": {
+        private ["_mre"];
+
+        _mre == "";
+
+        {
+            if(_x in (items player)) exitwith {
+                _mre = _x;
+            };
+        } forEach GVAR(food_items);
+
+        if(!(_mre == "")) then {
```

Comment:
if (not is equal)? How about if (is not equal)?
`if (_mre != "")`

Reply:
Done